addons/web/static/tests/chrome/action_manager_tests.js

```diff
@@ -1847,12 +1847,19 @@ QUnit.module('ActionManager', {
     });
 
     QUnit.test('state is pushed for client actions', async function (assert) {
-        assert.expect(2);
+        assert.expect(3);
```

Comment:
I could have isolated this in another test, but it would be so close to this one that I don't see the point...